ITsisse/week7.txt

```diff
@@ -50,7 +50,7 @@ American Standard Code for Information Interchange. 7 bitine tähemärkide tabel
 Proovis ühtsustada kõiki erinevaid tähtede kodeerimis viise, mis olid eri maailma osades tekkinud. Üle 128 000 sümboli. Eri kodeerimisviisid: UTF-8, UTF-16, UTF-32. UTF-8 on pmst ASCII, 0-127 ongi ASCII. UTF-16 saab enamusi levinumaid sümboleid. UTF-32 saab kõiki.
 
 18: UTF-8
-UTF-8 on pmst ASCII, 0-127 ongi ASCII. UTF-16 saab enamusi levinumaid sümboleid. UTF-32 saab kõiki.
```

Comment:
UTF-8 is a LOT more than just ASCII part, tho out of the 3 here its the only one backwards compatible with ASCII